script/install-sketchtool

```diff
@@ -1,19 +0,0 @@
-#!/bin/bash
```

Comment:
We should remove `script/install-sketchtool` from `script/bootstrap` also.

Reply:
Whoops, my bad. Done!